lib/Jsrt/ChakraCommon.h

```diff
@@ -500,7 +500,12 @@ typedef UINT32 DWORD;
         /// <summary>
         ///     Specified script is internal and non-user code. Hidden from debugger
         /// </summary>
-        JsParseScriptAttributeLibraryCode = 0x1
+        JsParseScriptAttributeLibraryCode = 0x1,
+        /// <summary>
+        ///     ChakraCore assumes ExternalArrayBuffer is Utf8 by default.
+        ///     This one needs to be set for Utf16
+        /// </summary>
+        JsParseScriptAttributeUtf16Encoding = 0x2,
```

Comment:
>JsParseScriptAttributeUtf16Encoding [](start = 8, length = 35)

The comment is clear, but users may possibly be confused by the name and think this flag always apply? Should we explicitly have "ArrayBuffer" in the flag name? Or not?